packages/manager/src/queries/types.ts

```diff
@@ -54,6 +54,6 @@ export const useTypeQuery = (type: string, enabled = true) => {
     queryKey: specificTypesQueryKey(type),
     queryFn: () => getType(type),
     ...queryPresets.oneTimeFetch,
-    enabled,
+    enabled: enabled && type !== '',
```

Comment:
Do we need to check for `type` being `null` as well/instead? That's what the linodes with no plans are coming back as having

Reply:
Yeah, we could but Typescript won't allow a null or undefined value to be passed as `type` here. 

Should I still make the check?

```suggestion
    enabled: enabled && Boolean(type),
```

Reply:
Gotcha, I think it'd be good just for consistency/clarity